bundles/org.openhab.binding.saicismart/README.md

```diff
@@ -0,0 +1,82 @@
+# SAICiSMART Binding
+
+OpenHAB binding to the SAIC-API used by MG cars (MG4, MG5 EV, MG ZSV...)
+
+It enables iSMART users to get battery status and other data from their cars. 
+They can also pre-heat their cars by turning ON the AC.
+
+Based on the work done here: https://github.com/SAIC-iSmart-API
+
+## Supported Things
+
+European iSMART accounts and vehicles.
+
+- `account`: An iSMART Account
```

Comment:
If i understand this setup, the account is a bridge and the vehicle is a thing. Might be wise to explicilty list that here

Reply:
Modified the text to make this clear.